File: SDKs/Aspose.Tasks_Cloud_SDK_for_Java/src/main/java/com/aspose/tasks/model/ExtendedAttribute.java
package com.aspose.tasks.model;

import com.aspose.tasks.model.TimeUnitType;
public class ExtendedAttribute {
  private String FieldId = null;
  private String Value = null;
  private String ValueGuid = null;
  private TimeUnitType DurationFormat = null;
  /**
	 * getFieldId
	 * Gets String
	 * @return FieldId
	 */
  public String getFieldId() {
    return FieldId;
  }

	/**
	 * setFieldId
	 * Sets String
	 * @param FieldId String
	 */
  public void setFieldId(String FieldId) {
    this.FieldId = FieldId;
  }

  /**
	 * getValue
	 * Gets String
	 * @return Value
	 */
  public String getValue() {
    return Value;
  }

	/**
	 * setValue
	 * Sets String
	 * @param Value String
	 */
  public void setValue(String Value) {
    this.Value = Value;
  }

  /**
	 * getValueGuid
	 * Gets String
	 * @return ValueGuid
	 */
  public String getValueGuid() {
    return ValueGuid;
  }

	/**
	 * setValueGuid
	 * Sets String
	 * @param ValueGuid String
	 */
  public void setValueGuid(String ValueGuid) {
    this.ValueGuid = ValueGuid;
  }

  /**
	 * getDurationFormat
	 * Gets TimeUnitType
	 * @return DurationFormat
	 */
  public TimeUnitType getDurationFormat() {
    return DurationFormat;
  }

	/**
	 * setDurationFormat
	 * Sets TimeUnitType
	 * @param DurationFormat TimeUnitType
	 */
  public void setDurationFormat(TimeUnitType DurationFormat) {
    this.DurationFormat = DurationFormat;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ExtendedAttribute {\n");
    sb.append("  FieldId: ").append(FieldId).append("\n");
    sb.append("  Value: ").append(Value).append("\n");
    sb.append("  ValueGuid: ").append(ValueGuid).append("\n");
    sb.append("  DurationFormat: ").append(DurationFormat).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Tasks_Cloud_SDK_for_CSharp/src/Com/Aspose/Tasks/Model/ExtendedAttribute.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model {
  public class ExtendedAttribute {
    public string FieldId { get; set; }

    public string Value { get; set; }

    public string ValueGuid { get; set; }

    public TimeUnitType DurationFormat { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ExtendedAttribute {\n");
      sb.Append("  FieldId: ").Append(FieldId).Append("\n");
      sb.Append("  Value: ").Append(Value).Append("\n");
      sb.Append("  ValueGuid: ").Append(ValueGuid).Append("\n");
      sb.Append("  DurationFormat: ").Append(DurationFormat).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
